### PR TITLE
adds the kinesis+ module

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

original pr: https://github.com/tgstation/tgstation/pull/65823
documents, cleans up some code in kinesis code and adds the ability for it to grab mobs through a var edit
adds a kinesis+ module that can grab mobs, gives it to chrono legionnaires and admin modsuits

## Why It's Good For The Game

image

## Changelog

:cl: Fikou, unit0016
add: Chrono Legionnaire and Admin MODsuits now have the ability to hold mobs with kinesis.
/:cl:

### CODE REVIEW

**Comment by CRITAWAKETS** [comment]
assistants aren't supposed to have guns

**Comment by Shadowflame909** [comment]
we must give abductors purchasable modsuits

So we can give them this module! Alien probe reference.

**Comment by Valorium-RB** [comment]
> we must give abductors purchasable modsuits
> 
> So we can give them this module! Alien probe reference.

Actually a good idea. When was the last time abductors got new content?

File: code/modules/mod/modules/module_kinesis.dm [hunk 1]
```diff
@@ -15,27 +15,32 @@
...
 	var/datum/looping_sound/gravgen/kinesis/soundloop
+	/// The cooldown between us hitting objects with kinesis.
 	COOLDOWN_DECLARE(hit_cooldown)
 
 /obj/item/mod/module/anomaly_locked/kinesis/Initialize(mapload)
 	. = ..()
 	soundloop = new(src)
 
 /obj/item/mod/module/anomaly_locked/kinesis/Destroy()
-	if(grabbed_atom)
```
Comment (SuperNovaa41) [inline code/modules/mod/modules/module_kinesis.dm:33]: is none of this needed anymore?

Comment (Fikou) [reply code/modules/mod/modules/module_kinesis.dm:33]: yeah, its handled by another piece of code

**Pull request merged**